test/wallaby/phantom/log_store_test.exs

```diff
@@ -9,12 +9,21 @@ defmodule Wallaby.Driver.LogStoreTest do
 
   @session "123abc"
 
+  setup do
```

Comment:
Not sure why are we making this change? Seems unrelated to hover

Reply:
See [previous comment](https://github.com/keathley/wallaby/pull/435#discussion_r304444908)

Reply:
Looks good, but can we keep the call the same? I think LogStore being an Agent is an implementation detail and we can keep LogStore.start_link here

Reply:
The call to `LogStore.start_link` was actually silently (silent because it wasn't matching on anything) failing because `LogStore` is actually started under a supervisor when the app starts up. 

The state needs to be cleared somehow between tests or the implementation of LogStore needs to change to take a name so it can be started independently while under test.

In general I agree with you though. I will figure something out.